### ISSUE TITLE
FFI Bindings

### ISSUE THREAD

**Issue by thesimplekid** [cashubtc/cdk#160]

**Comment by dluvian** [comment]
Kotlin bindings would be based

**Comment by thesimplekid** [comment]
> Kotlin bindings would be based

Yes kotlin bindings will be included as part of the Kotlin [uniffi bindings](https://github.com/mozilla/uniffi-rs?tab=readme-ov-file)
